argus/action_manager/windows.py

```diff
@@ -179,12 +180,34 @@ def git_clone(self, repo_url, location):
         :param location:
             Specific location on the instance.
         """
-        LOG.info("Cloning from %s to %s", repo_url, location)
-        cmd = "git clone {} {}".format(repo_url, location)
-        self._client.run_command_with_retry(cmd,
-                                            count=util.RETRY_COUNT,
-                                            delay=util.RETRY_DELAY,
-                                            command_type=util.CMD)
+        LOG.info('Trying to clone from %s to "%s"...', repo_url, location)
+        cmd = 'git clone {} "{}"'.format(repo_url, location)
+
+        for _ in range(util.RETRY_COUNT):
+            try:
+                if self.exists(location) and self.is_dir(location):
+                    self.rmdir(location)
+
+                LOG.info("Cloning the repository...")
+                _, stderr, exit_code = self._client.run_remote_cmd(
+                    cmd, command_type=util.CMD)
+
+                if stderr and exit_code:
+                    LOG.debug("Failed to clone the repository: %s "
+                              "(return code: %s)" % (stderr, exit_code))
+                else:
+                    LOG.info("The repository was cloned successfully!")
+                    break
+            except (socket.error, winrm_exceptions.WinRMError,
+                    requests.RequestException) as exc:
+                LOG.debug("Failed to clone the repository: %s" % exc)
```

Comment:
Ditto, use the same style `LOG.info('Trying to clone from %s to "%s"...', repo_url, location)` or `LOG.debug("Failed to clone the repository: %s" % exc)`